File: Map/src/app/shared/DanhMucLoaiGiong.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { DanhMucLoaiGiong } from 'src/app/shared/DanhMucLoaiGiong.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class DanhMucLoaiGiongService extends BaseService{
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "DanhMucLoaiGiong";
    }
}

File: Service/Implement/DanhMucLoaiGiongService.cs
namespace Service.Implement
{
    public class DanhMucLoaiGiongService : BaseService<DanhMucLoaiGiong, IDanhMucLoaiGiongRepository>
    , IDanhMucLoaiGiongService
    {
    private readonly IDanhMucLoaiGiongRepository _DanhMucLoaiGiongRepository;
    public DanhMucLoaiGiongService(IDanhMucLoaiGiongRepository DanhMucLoaiGiongRepository) : base(DanhMucLoaiGiongRepository)
    {
    _DanhMucLoaiGiongRepository = DanhMucLoaiGiongRepository;
    }
    }
    }

